### PR TITLE
Some tweak for messaging around credentials

### PR DESCRIPTION
**Pull request by keiko713**
I used these commands several times today and noticed that we want to tweak a bit.

### CODE REVIEW

File: commands/credentials/create.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function * run (context, heroku) {
   yield cli.action(`Creating credential ${cli.color.cmd(flags.name)}`, co(function * () {
     yield heroku.post(`/postgres/v0/databases/${db.name}/credentials`, {host: host(db), body: data})
   }))
-  let attachCmd = `heroku addons:attach ${db.name} --credential ${flags.name} -a app_name`
+  let attachCmd = `heroku addons:attach ${db.name} --credential ${flags.name} -a ${app}`
```
Comment (msakrejda) [inline commands/credentials/create.js:22]: Nice catch.

**Review by msakrejda — APPROVED** [review]
+1

**Review by camillebaldock — APPROVED** [review]
nice, thanks!

**Comment by keiko713** [comment]
Uhmm... who is exactly gonna ship this?

**Comment by camillebaldock** [comment]
@keiko713 click merge, and once a few more fixes are in, i'll get a release out :)

**Pull request merged**